src/mlpack/methods/reinforcement_learning/q_networks/simple_dqn.hpp

```diff
@@ -21,17 +21,15 @@
 namespace mlpack {
 namespace rl {
 
-using namespace mlpack::ann;
-
 /**
  * @tparam OutputLayerType The output layer type of the network.
  * @tparam InitType The initialization type used for the network.
  * @tparam NetworkType The type of network used for simple dqn.
  */
 template<
-  typename OutputLayerType = MeanSquaredError<>,
-  typename InitType = GaussianInitialization,
-  typename NetworkType = FFN<OutputLayerType, InitType>
+  typename OutputLayerType = mlpack::ann::MeanSquaredError<>,
+  typename InitType = mlpack::ann::GaussianInitialization,
+  typename NetworkType = mlpack::ann::FFN<OutputLayerType, InitType>
```

Comment:
```suggestion
  typename OutputLayerType = ann::MeanSquaredError<>,
  typename InitType = ann::GaussianInitialization,
  typename NetworkType = ann::FFN<OutputLayerType, InitType>
```
(Another file where maybe we can do the same thing.)